blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -40,9 +40,11 @@ export default class Pipelines extends Component {
                             { organization && ' / ' }
                             { organization && orgLink }
                         </h1>
-                        <a target="_blank" className="btn-secondary inverse" href={newJobUrl}>
-                            New Pipeline
-                        </a>
+                        <Extensions.Renderer extensionPoint="jenkins.pipeline.create.action">
```

Comment:
right - seems reasonably. The 'additive' ness would happen if there were multiple things for the one extension point (ie you could put the "built in" behavior as an extension if you wanted it to be additive, right?)